### DISCUSSION TITLE
Type of labeling format used

### DISCUSSION THREAD

**Discussion by OrranaLhaynher** [tecoholic/ner-annotator#46] [General]
Hello, my question is about the type of labeling format used. Was it IOB, IOB2, IOBES, or another type of format?

**Comment by tecoholic** [comment]
@OrranaLhaynher Oh, I didn't know such formats even existed. As there is no automation involved, the labels are user created. So you can pretty much use any format you want to use I think. Sorry, it this is not very helpful.
